### ISSUE TITLE
Layers not available in GGIS viewers

### ISSUE THREAD

**Issue by ArnaudIGRAC** [kartoza/IGRAC-GGIS#498]
Layers are not avaiable in GGIS viewers! Check for instance 
https://ggis.un-igrac.org/view/tba
![image](https://user-images.githubusercontent.com/64771599/185893441-c2d193a3-3da9-4d20-bc94-1cf21acb2f63.png)

https://ggis.un-igrac.org/view/marportal
![image](https://user-images.githubusercontent.com/64771599/185893497-ec4367ed-fe2c-4236-ac38-c21698325c5f.png)

**Comment by ArnaudIGRAC** [comment]
the layers are back, have you done anything?

**Comment by NyakudyaA** [comment]
I think maybe you tried to access the layers when the pod was either restarting itself. We didn't get any error indication.

**Comment by meomancer** [comment]
Hi @ArnaudIGRAC 

I didn't do anything, it seems the geoserver was restarting when you accessing it, as @NyakudyaA said

**Comment by ArnaudIGRAC** [comment]
alright, thank you for your quick feedback!

**Comment by ArnaudIGRAC** [comment]
Hi Irwan, I have that same issue again. Layers have been unavailable for 15 minutes now

**Comment by ArnaudIGRAC** [comment]
I have this issue again. Never occurred in the past. There is something wrong here, to be addressed.

**Comment by ArnaudIGRAC** [comment]
We have that same issue again

**Comment by ArnaudIGRAC** [comment]
Yes, probably. But it happens quite often, and it last for several minutes, during which the GGIS cannot be used.

**Comment by ArnaudIGRAC** [comment]
You might also want to check the SADC-GIP, there is the same issue there, for several days now.

**Comment by ArnaudIGRAC** [comment]
I will close this issue for now, hopefully it won't happen again
